log-configs/template-demo.yaml

```diff
@@ -8,3 +8,9 @@ loggerTree:
   files:
     - {{file}}.pub
```

Comment:
we can actually remove the *.pub file as the JSON log will replace it

Reply:
Done.